components/map/Map.tsx

```diff
@@ -115,14 +130,30 @@ const Map = ({ onBottomSheetContentChange }: Props) => {
             animated
           />
         )}
-        {udrData && (
-          <ShapeSource id="udrSource" shape={udrData}>
+        {udrDataByPrice.regular?.features?.length > 0 && (
```

Comment:
Eslint should complain to this condition, something like "you cannot compare undefined". Isn't it complaining?

Reply:
Or maybe those question marks are not needed?

Reply:
no 😕

Reply:
I reverted the changes to eslint config and now the error is there 👍